src/pages/Login.js

```diff
@@ -51,26 +53,40 @@ export default function Login() {
           {({ isSubmitting }) => (
             <FormStyled>
               <LoginStyled>Login</LoginStyled>
-              <FieldStyled type="email" name="email" placeholder="E-Mail" />
+              <FieldStyled
+                type="email"
+                name="email"
+                placeholder="E-Mail"
+                value={profile.email}
+                onChange={handleChange}
+              />
               <ErrorMessage name="email" component="div" className="error" />
               <FieldStyled
                 type="password"
                 name="password"
                 placeholder="Passwort"
+                value={profile.password}
+                onChange={handleChange}
```

Comment:
Ich würde die Funktion evtl etwas deutlicher benennen zB saveInput oder soetwas, damit man weiß, was passiert.

Reply:
hier auch.. wenn ich die funktion umbenennepassiert nichts mehr in firebase :(